Sources/BraintreePaymentFlow/BTLocalPaymentRequest.swift

```diff
@@ -167,7 +174,7 @@ extension BTLocalPaymentRequest: BTPaymentFlowRequestDelegate {
             apiClient.post("v1/local_payments/create", parameters: params) { body, response, error in
                 if let error {
                     if (error as NSError).code == BTCoreConstants.networkConnectionLostCode {
-                        apiClient.sendAnalyticsEvent("ios.local-payment-methods.network-connection.failure")
+                        apiClient.sendAnalyticsEvent(BTPaymentFlowAnalytics.paymentNetworkConnectionLost)
```

Comment:
This event is missing from our event spreadsheet. Can you add it?

Also, it is missing from SEPA & 3DS as well.

Reply:
So we do want this for all of the modules? I was following the spreadsheet for other modules.

Reply:
I updated the spreadsheet to include networkLost events for all modules!

Reply:
Here is the Jira ticket for adding networkConnectionLost event for all modules: https://paypal.atlassian.net/browse/DTBTSDK-2711